packages/gatsby-theme-api-docs/src/components/resource/method/method.js

```diff
@@ -58,69 +59,73 @@ const Method = ({
   });
 
   return (
-    <SpacingsStack scale="s">
```

Comment:
A noisy looking change in GitHub that is in fact only wrapping the existing code into a `<FullWidthContainer>`

Reply:
In case you didn't know, there is an option to hide whitespace changes. This then looks like this:

<img width="1537" alt="image" src="https://user-images.githubusercontent.com/1110551/123086585-e1abab00-d423-11eb-96fb-66f922dc53c6.png">